migrations/Version202009171027563772_taoLti.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoLti\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\update\OntologyUpdater;
+
+final class Version202009171027563772_taoLti extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
```

Comment:
I am not sure we need 2 similar migrations at the end. So when we merge to develop users that install this extensions will have models synched twice.

Reply:
Totally agree with you, before merging integration branch into develop all created migrations should be revisited

Reply:
Yes, you are right we can manage it in the final merge :) Thanks for that!